### ISSUE TITLE
关于loss的计算

### ISSUE THREAD

**Issue by 408550969** [yzd-v/FGD#39]
你好，非常感谢你的分享，我有一点不是很确定，最终的loss是由原检测器loss+每个fpn输出的特征层的FeatureLoss吗？比如fpn有三层，最终的loss是如下这样吗？
for i in stu_features:
         distilloss=FeatureLoss(stu_features[i],tea_features[i],gt_bboxes,img_metas)
         loss= loss + distilloss

**Comment by yzd-v** [comment]
是这样计算的

**Comment by 408550969** [comment]
谢谢
